src/test/resources/tema7catalinaz/groups.xml

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd" >
```

Comment:
Please rename folder `tema7catalinaz ` to `catalinazoldi`. Thanks!
![image](https://user-images.githubusercontent.com/18215854/219291814-4d21caed-5532-4bee-984e-0cf8969fa2aa.png)